docs/features/fixed-window-rate-limiter.md

```diff
@@ -0,0 +1,24 @@
+# Fixed Window Rate Limiter
+
+A "rate limiter" controls how frequently some event (e.g. an HTTP request or a login attempt) is allowed to happen. 
+Rate limiting is commonly used as a defensive measure to protect services from excessive use (intended or not) 
+and maintain their availability. 
+It's also useful to control your internal or outbound processes (e.g. limit the number of simultaneously processed messages).
+
+Currently, only Fixed Window Rate Limiter is implemented on this application This is the simplest technique and 
+it's based on setting a limit for a given interval of time (e.g. 5,000 requests per hour or 3 login attempts every 
+15 minutes).
+
+## Table of Contents
+
+- [Related environment variables](#related-environment-variables)
+
+### Related environment variables
+
+| Variable | Description |
+| -----------------------------------  |:--------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `RATE_LIMITER_STORAGE_DSN`           | The data source name of the rate limiter storage. Ex: `redis://localhost`                                                                                           |
+| `RATE_LIMITER_FIXED_WINDOW_ROUTES`   | A list of route names (defined in [routes.yaml](../../config/routes.yaml)) separeted by comma. Ex: `"getAccessToken,refreshAccessToken,bulkCreateUsersAssignments"` |
+| `RATE_LIMITER_FIXED_WINDOW_LIMIT`    | Number of requests that can be done during the given interval                                                                                                       |
+| `RATE_LIMITER_FIXED_WINDOW_INTERVAL` | The value of the interval option must be a number followed by any of the units accepted by the PHP date relative formats (e.g. 3 seconds, 10 hours, 1 day, etc.)    |
+| `LOCK_DSN`                           | The data source name of the LOCK storage. Ex: `redis://localhost`                                                                                                   |
```

Comment:
It would be nice to add a little bit more explanation to these variables. Why do we need two storages here? What will be stored exactly there?

Also I don't think that it's convenient to specify all the different routes one by one. It's good to have this flexibility, but the default value could be `all` or something like this. What do you think?

Reply:
Also please add these variables to devops documentation as well.

Reply:
What do you guys think:
```
RATE_LIMITER_FIXED_WINDOW_ROUTES="" //disable the feature
RATE_LIMITER_FIXED_WINDOW_ROUTES="*" //enable for all routes (if we use "all" there is a remote possibility that we may create a route with the same name in the future).

Reply:
`*` is better indeed.